sql/17.0/extra/returning.sql

```diff
@@ -0,0 +1,146 @@
+--Testcase 23:
+DELETE FROM "type_FLOAT" RETURNING *;
+--Testcase 24:
+SELECT * FROM "type_DOUBLE";
+--Testcase 25:
+DELETE FROM "type_DOUBLE" RETURNING *;
+--
+set datestyle=ISO;
+--Testcase 26:
+SELECT * FROM "type_TIMESTAMP";
+--Testcase 27:
+DELETE FROM "type_TIMESTAMP" RETURNING *;
+--Testcase 28:
+SELECT * FROM "type_BLOB";
+--Testcase 29:
+DELETE FROM "type_BLOB" RETURNING *;
+--Testcase 30:
+ALTER TABLE "type_UUID" ALTER COLUMN "i" OPTIONS (ADD key 'true');
+--Testcase 31:
+DELETE FROM "type_UUID" RETURNING *;
+--Testcase 32:
+DELETE FROM "type_BITSTRING" RETURNING *;
+
+--Testcase 33:
+SELECT * FROM typetest;
+--Testcase 34:
+INSERT INTO "type_STRING"(col) VALUES ('string') ON CONFLICT DO NOTHING RETURNING *;
+--Testcase 35:
+ EXPLAIN (VERBOSE, COSTS OFF)
+INSERT INTO "type_STRING"(col) VALUES ('string') RETURNING *;
+--Testcase 36:
+INSERT INTO "type_BYTE"(col) VALUES ('c') RETURNING *;
+--Testcase 37:
+INSERT INTO "type_SINT"(col) VALUES (32767) RETURNING *;
+--Testcase 38:
+INSERT INTO "type_SINT"(col) VALUES (-32768) RETURNING *;
+--Testcase 39:
+INSERT INTO "type_BINT"(col) VALUES (9223372036854775807) RETURNING *;
+--Testcase 40:
+INSERT INTO "type_BINT"(col) VALUES (-9223372036854775808) RETURNING *;
+--Testcase 41:
+INSERT INTO "type_INTEGER"(col) VALUES (9223372036854775807) RETURNING *;
+
+--Testcase 42:
+INSERT INTO "type_FLOAT"(col) VALUES (3.1415) RETURNING *;
+--Testcase 43:
+INSERT INTO "type_DOUBLE"(col) VALUES (3.14159265) RETURNING *;
+--Testcase 44:
+INSERT INTO "type_TIMESTAMP" VALUES ('2017.11.06 12:34:56.789', '2017.11.06') RETURNING *;
+--Testcase 45:
+INSERT INTO "type_TIMESTAMP" VALUES ('2017.11.06 1:3:0', '2017.11.07') RETURNING *;
+--Testcase 46:
+INSERT INTO "type_BLOB"(col) VALUES (bytea('\xDEADBEEF')) RETURNING *;
+--Testcase 47:
+INSERT INTO typetest VALUES(1,'a', 'b', 'c','2017.11.06 12:34:56.789', '2017.11.06 12:34:56.789' ) RETURNING *;
+
+--Testcase 48:
+SELECT * FROM "type_DATE";
+--Testcase 49:
+INSERT INTO "type_DATE"(col) VALUES ('2021.02.23') RETURNING col;
+--Testcase 50:
+INSERT INTO "type_DATE"(col) VALUES ('2021/03/08') RETURNING col;
+--Testcase 51:
+INSERT INTO "type_DATE"(col) VALUES ('9999-12-30') RETURNING col;
+--Testcase 52:
+SELECT * FROM "type_DATE";
+
+--Testcase 53:
+SELECT * FROM "type_TIME";
+--Testcase 54:
+INSERT INTO "type_TIME"(col) VALUES ('01:23:45') RETURNING col;
+--Testcase 55:
+INSERT INTO "type_TIME"(col) VALUES ('01:23:45.6789') RETURNING col;
+--Testcase 56:
+SELECT * FROM "type_TIME";
+
+--Testcase 57:
+CREATE FOREIGN TABLE type_JSON(col JSON OPTIONS (key 'true')) SERVER sqlite_svr OPTIONS (table 'type_TEXT');
+--Testcase 58
+SELECT * FROM type_JSON;
+--Testcase 59:
+INSERT INTO type_JSON(col) VALUES ('[1, 2, "foo", null]') RETURNING *;
+--Testcase 60:
+INSERT INTO type_JSON(col) VALUES ('{"bar": "baz", "balance": 7.77, "active": false}'::json) RETURNING *;
+--Testcase 61
+SELECT * FROM type_JSON;
+--Testcase 62
+DELETE FROM type_JSON RETURNING *;
+
```

Comment:
According to the specification https://www.sqlite.org/lang_returning.html, could you please add more test cases regarding RETURNING to verify the following points?

INSERT/DELETE/UPDATE combine RETURNING
1) RETURNING expr
2) RETURNING expr1, expr2,...
3) RETURNING expr AS column-alias
4) RETURNING expr1 AS column-alias1, expr2 AS column-alias2,...
5) The RETURNING clause does not report any additional database changes caused by [foreign key constraints](https://www.sqlite.org/foreignkeys.html) or [triggers](https://www.sqlite.org/lang_createtrigger.html).
6) The output order for the RETURNING rows is arbitrary and is not necessarily related to the order in which the rows were processed internally.
7) 07 items of Limitations And Caveats

Reply:
Will TC 60 ..100 enough?